src/schema.js

```diff
@@ -202,16 +214,31 @@ export default class DataStructureGenerator {
         element.description = new StringElement(schema.description);
       }
 
-      if (schema.default !== undefined && !_.isArray(schema.default) &&
-          !_.isObject(schema.default)) {
+      let def = schema.default;
+
+      if (def !== undefined && !_.isArray(def) && !_.isObject(def)) {
         // TODO Support defaults for arrays and objects
```

Comment:
I think we should be able to remove this TODO and `isArray`/`isObject` check now, but we can do this in separate PR.